udfs/flex/text/levenshtein.md

```
@@ -0,0 +1,68 @@
+# text.levenshtein
+
+## Description
+Computes the Levenshtein edit distance between two strings. The edit distance is the minimum number of single-character edits (insertions, deletions, or substitutions) required to change one string into another. This is useful for fuzzy string matching, spell checking, and finding similar records.
+
+## Syntax
+```cypher
+flex.text.levenshtein(string1, string2)
+```
+
+## Parameters
+| Parameter | Type | Required | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)